src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/price/Price.kt

```diff
@@ -50,6 +50,15 @@ data class Price(
   @Column(name = "effective_year", nullable = false)
   val effectiveYear: Int
 ) {
+
+  init {
+    failOnZeroOrLessPrice()
+  }
+
+  private fun failOnZeroOrLessPrice() {
+    if (priceInPence < 1) throw IllegalArgumentException("Price in pence must be greater than zero.")
```

Comment:
Also adding guard at the Domain level for belt and braces.  Worst case here (if anything is missed in future) would end up being directed to the general error page but at least it would not allow an invalid price to be saved.